pvlib/clearsky.py

```diff
@@ -633,15 +633,16 @@ def _calc_stats(data, samples_per_window, sample_interval, H):
     # shift to get forward difference, .diff() is backward difference instead
     data_diff = data.diff().shift(-1)
     data_slope = data_diff / sample_interval
-    data_slope_nstd = _slope_nstd_windowed(data, H, samples_per_window)
+    data_slope_nstd = _slope_nstd_windowed(data, H, samples_per_window,
+                                           sample_interval)
     data_slope_nstd = data_slope_nstd
 
     return data_mean, data_max, data_slope_nstd, data_slope
 
 
-def _slope_nstd_windowed(data, H, samples_per_window):
+def _slope_nstd_windowed(data, H, samples_per_window, sample_interval):
     with np.errstate(divide='ignore', invalid='ignore'):
-        raw = np.diff(data)
+        raw = np.diff(data) / sample_interval
```

Comment:
Somehow I failed to include this comment earlier:  is `raw` here equivalent to `data_slope.values[:-1]`?  If so there might be some modest speedup to be had by passing that in to avoid calculating slope twice.